docs/extras/building_vtk.rst

```diff
@@ -2,78 +2,256 @@
+Kitware provides Python wheels for VTK at `PyPI VTK
+<https://pypi.org/project/vtk/>`_, but there are situations where you
 may need to build VTK from source (e.g. new release of Python, EGL
 rendering, additional features, etc).  As ``pyvista`` does not provide
 ``vtk``, you will have to build it manually.
 
+Should you need a prebuilt wheel, a variety of prebuilt wheels
+(including ones with EGL) can be found at `pyvista Release 0.31.0
+<https://github.com/pyvista/pyvista/releases/tag/0.31.0>`_.  This
+includes wheels for Raspberry Pi (64-bit OS), Python 3.9 Linux, and
+EGL support (``arch64`` and ``x86_64``).
+
 
 Wheels on Linux and Mac OS
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Building VTK from source on Linux is fairly easy.  Using the default
-build settings, build a Python wheel of VTK using ``ninja`` using the following script.  This script assumes Python 3.9, but you can use any modern Python version.
-For some additional useful options, see the `conda-forge recipe <https://github.com/conda-forge/vtk-feedstock/blob/master/recipe/build.sh>`__.
-Most of the ones below are designed to reduce the build time and resulting wheel size.
+Building VTK from source on Linux and MacOS is fairly easy.  Using the
+default build settings, build a Python wheel of VTK using ``ninja``
+using the following script.  This script uses system python3, but you
+can use any modern Python version.  For some additional useful
+options, see the `conda-forge recipe
+<https://github.com/conda-forge/vtk-feedstock/blob/master/recipe/build.sh>`__.
+Most of the ones below are designed to reduce the build time and
+resulting wheel size.
 
 .. code-block:: bash
 
+    #!/bin/bash
+
+    # install build dependencies (Linux/Debian)
+    sudo apt-get install ninja-build cmake libgl1-mesa-dev python3-dev -y
+
+    # Linux/CentOS
+    sudo yum install epel-release -y
+    sudo yum install ninja-build cmake mesa-libGL-devel mesa-libGLU-devel -y
+
     git clone https://github.com/Kitware/VTK
     cd VTK
 
-    mkdir build
-    cd build
-    PYBIN=/usr/bin/python3.9
-    cmake -GNinja -DCMAKE_BUILD_TYPE=Release -DVTK_BUILD_TESTING=OFF \
-        -DVTK_BUILD_DOCUMENTATION=OFF -DVTK_BUILD_EXAMPLES=OFF -DVTK_DATA_EXCLUDE_FROM_ALL:BOOL=ON \
-        -DVTK_MODULE_ENABLE_VTK_PythonInterpreter:STRING=NO \
-        -DVTK_WHEEL_BUILD=ON -DVTK_PYTHON_VERSION=3 -DVTK_WRAP_PYTHON=ON \
-        -DPython3_EXECUTABLE=$PYBIN ../
+    PYBIN=/usr/bin/python3
+    cmake -GNinja \
+          -DCMAKE_BUILD_TYPE=Release \
+          -DVTK_BUILD_TESTING=OFF \
+          -DVTK_BUILD_DOCUMENTATION=OFF \
+          -DVTK_BUILD_EXAMPLES=OFF \
+          -DVTK_DATA_EXCLUDE_FROM_ALL:BOOL=ON \
+          -DVTK_MODULE_ENABLE_VTK_PythonInterpreter:STRING=NO \
+          -DVTK_WHEEL_BUILD=ON \
+          -DVTK_PYTHON_VERSION=3 \
+          -DVTK_WRAP_PYTHON=ON \
+          -DVTK_OPENGL_HAS_EGL=False \
+          -DPython3_EXECUTABLE=$PYBIN ../
     ninja
     $PYBIN setup.py bdist_wheel
```

Comment:
If I recall correctly the `bdist_wheel` directive needs the `wheel` package to be pip installed. We might want to mention that somewhere.

We might also want to mention in a comment that this creates the wheels in `dist/`, to match the comment on the next line.